docs/root/api/api_supported_versions.rst

```diff
@@ -0,0 +1,18 @@
+.. _api_supported_versions:
+
+Suported API versions
+=====================
+
+Envoy's APIs follow a `versioning scheme
+<https://github.com/envoyproxy/envoy/blob/master/api/API_VERSIONING.md>`_ in which Envoy supports
```

Comment:
use a `:repo:` link here to snap the version of the file, same elsewhere if I missed anything like this.